frontend/src/views/supp-allowances/SupplementaryConfirmation.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-container class="mt-16">
+    <div class="mt-16">
+      <h2 class="mb-8 text-center">Supplementary Allowance Submitted</h2>
+      <h4 class="mb-8 text-center">Check your dashboard for updates on the progress of your application. We may reach out if we need more information</h4>
+
+      <v-row justify="space-around" no-gutters class="mt-10">
+        <AppBackButton id="back-home-button" :primary="false" width="250px" :to="{ name: 'home' }" align="center">Home</AppBackButton>
+        <AppButton id="view-applications-button" size="large" width="250px" :to="{ name: 'applications-history' }" align="center" class="mt-2">View Applications</AppButton>
+      </v-row>
+    </div>
+  </v-container>
+</template>
+
+<script>
+import AppButton from '@/components/ui/AppButton.vue'
+import AppBackButton from '@/components/ui/AppBackButton.vue'
+
+export default {
+  name: 'ApplicationConfirmationView',
+  components: { AppButton, AppBackButton },
+  computed: {},
+  created() {},
```

Comment:
can we remove computed and created if they are unused?

Reply:
done